frontend/src/run.js

```diff
@@ -460,6 +460,10 @@ export class Run extends React.Component {
         passed -= run.summary.skips;
```

Comment:
Once you've changed the backend to put the passed results into a "passes" field, you can update this to use that field if it exists.